docker-compose.yml

```diff
@@ -105,7 +105,8 @@ services:
       - esdata:/usr/share/elasticsearch/data
 
   redis:
-    image: redis
+    # The version of this Redis image should be kept synchronized with production.
```

Comment:
Would it be worth linking to what that is?
E.g. a link to https://github.com/mdn/infra/blob/1c842b5fb81b0eccf8624852e0dffca8ac13ab65/apps/mdn/mdn-aws/infra/modules/multi_region/redis/variables.tf#L41
Or, perhaps https://github.com/mdn/infra/blob/master/apps/mdn/mdn-aws/infra/modules/multi_region/redis/variables.tf in general. 
That way, it's easier to remember where to look that up.

Reply:
👍 I'll do that.